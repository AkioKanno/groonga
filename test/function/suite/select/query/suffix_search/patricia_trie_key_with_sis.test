table_create Names TABLE_PAT_KEY|KEY_WITH_SIS ShortText
column_create Names original COLUMN_SCALAR Bool
load --table Names
[
{"_key": "ひろゆき", "original": true},
{"_key": "まろゆき", "original": true},
{"_key": "ひろあき", "original": true},
{"_key": "ゆきひろ", "original": true}
]
select --table Names
select --table Names --query _key:$ゆき
