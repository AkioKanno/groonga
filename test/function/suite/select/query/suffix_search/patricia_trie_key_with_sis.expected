table_create Names TABLE_PAT_KEY|KEY_WITH_SIS ShortText
[[0,0.0,0.0],true]
column_create Names original COLUMN_SCALAR Bool
[[0,0.0,0.0],true]
load --table Names
[
{"_key": "ひろゆき", "original": true},
{"_key": "まろゆき", "original": true},
{"_key": "ひろあき", "original": true},
{"_key": "ゆきひろ", "original": true}
]
[[0,0.0,0.0],4]
select --table Names --query _key:$ゆき
[
  [
    0,
    0.0,
    0.0
  ],
  [
    [
      [
        4
      ],
      [
        [
          "_id",
          "UInt32"
        ],
        [
          "_key",
          "ShortText"
        ],
        [
          "original",
          "Bool"
        ]
      ],
      [
        3,
        "ゆき",
        false
      ],
      [
        2,
        "ろゆき",
        false
      ],
      [
        5,
        "まろゆき",
        true
      ],
      [
        1,
        "ひろゆき",
        true
      ]
    ]
  ]
]
